src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -317,154 +154,62 @@ public void should_succeed_and_return_204_when_a_task_that_was_already_claimed_a
     }
 
     @Test
-    public void should_return_a_404_if_task_does_not_exist_with_completion_options_assign_and_complete_true() {
-        String nonExistentTaskId = "00000000-0000-0000-0000-000000000000";
-
-        common.setupCFTOrganisationalRoleAssignmentForWA(caseworkerCredentials.getHeaders());
-
-        Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED,
-            nonExistentTaskId,
-            new CompleteTaskRequest(new CompletionOptions(true)),
-            caseworkerCredentials.getHeaders()
-        );
-
-        result.then().assertThat()
-            .statusCode(HttpStatus.NOT_FOUND.value())
-            .and()
-            .contentType(APPLICATION_PROBLEM_JSON_VALUE)
-            .body("type",
-                equalTo("https://github.com/hmcts/wa-task-management-api/problem/task-not-found-error"))
-            .body("title", equalTo("Task Not Found Error"))
-            .body("status", equalTo(HttpStatus.NOT_FOUND.value()))
-            .body("detail", equalTo("Task Not Found Error: The task could not be found."));
-
-    }
-
-    @Test
-    public void should_return_a_204_when_completing_a_task_with_completion_options_assign_and_complete_true() {
```

Comment:
Is this in IT?

Reply:
Already added in 1878 pr.
https://github.com/hmcts/wa-task-management-api/pull/812/files#diff-f3e1c717e64a1f5bc400da9738643e29b8750206f06dc8bf86033f1f8fde1847L112